README.md

```diff
@@ -1,6 +1,22 @@
-# portfolio projects
+# Project Name
 
-> Portfolio, mobile version
+## portfolio projects
+
+> This is a website about showing projects that I have done.
+
+## Getting Started
```

Comment:
- [ ] Your portfolio is looking great, congrats on it! However, as the previous reviewer said, documentation is a very important part of every project and also a general requirement, please add some basic instructions to clone and run your project and any other relevant information for the visitor, you can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as an example.